casepro/cases/models.py

```diff
@@ -103,6 +103,16 @@ def __str__(self):
         return self.name
 
 
+class SystemUser(User):
+
+    @classmethod
+    def get_or_create(cls):
+        systemUsers = cls.objects.filter()
+        if len(systemUsers) > 0:
```

Comment:
Preference to `exists()` because that's super quick on postgres, `count()` still does a tally which is expensive in Postgresql. `exists()` is super quick because it just needs for the existence of 1. 
`len(queryset)` does a `queryset.count()` under the hood.